BirthdayCakeCandles.js

```diff
@@ -0,0 +1,6 @@
+function birthdayCakeCandles(candles) {
+    
+    let max = Math.max(...candles);
```

Comment:
There's no need to use js array restructuring here. You can totally pass in the given array (parameters) to the max method. It's because `Math.max` does not mutate the parameter `candles`. Or in other words, the parameter `candles` will be still the same after being passed in the max method.

JS array destructering will create another array which will cost memories.

I understand that you're trying to do 2 things:
- Find the max value
- Count the how many max values present in the list of given numbers (candles)

Finding the max value will iterate each item in the list then counting the max occurrences will do the same thing. I thought that it will cause a performance hit with a huge numbers of candles since your logic has to iterate all of the numbers in the list twice.

Please try to:
- Remove js array destructering
- Try to find max value and count the occurrences within ONE loop